userdocs/src/getting-started/application-addons.md

```diff
@@ -0,0 +1,36 @@
+To help manage the application lifecycle we provide addons that aim to make your life easier as a developer. While there aren't many of these available at the moment, we expect this to change as new requirements in the product teams surface.
+
+## Postgres
+
+It is now possible to create an [AWS RDS Postgres](https://aws.amazon.com/rds/postgresql/) database from the cluster declaration.  By adding the following section to your scaffolded `cluster.yaml` file:
+
+```yaml
+metadata:
+  environment: test # Just here to clarify the example
+databases:
+  postgres:
+    - name: dbtest
+      user: administrator
+      namespace: dbtest
+```
+
+Followed by applying the updated declaration with:
+
+```bash
+$ okctl apply cluster -f cluster.yaml
+```
+
+This will create a Postgres database in your AWS account together with a number of related resources:
+
+- A secrets manager secret for the administrator password
+- Lambda function that rotates the administrator password every 30 days
+- A security group that allows Pods that have that association to connect with the RDS Postgres database
```

Comment:
I just found out about the "Security groups for pods can't be used with pods deployed to Fargate."-thing, which I think is an important security detail. Could we maybe add an asterix or something that links to this information?